### ISSUE TITLE
no output with set_boxed_logger

### ISSUE THREAD

**Issue by bbigras** [rust-lang-nursery/log#261]
I'm probably doing something wrong but I don't understand why I don't get any output with this:

rustc 1.25.0-nightly (0c6091fbd 2018-02-04)
log 0.4.1

```rust
#[macro_use]
extern crate log;

use log::{Log, Metadata, Record};

struct MyLogger;

impl Log for MyLogger {
    fn enabled(&self, _metadata: &Metadata) -> bool {
        true
    }

    fn log(&self, record: &Record) {
        println!(">>> {} - {}", record.level(), record.args());
    }
    fn flush(&self) {}
}

fn main() {
    log::set_boxed_logger(Box::new(MyLogger {})).unwrap();

    error!("test");
}
```

**Comment by sfackler** [comment]
The max log level filter defaults to "off" - you'll just need to adjust that: `log::set_max_level(log::LevelFilter::Info)`.

**Comment by bbigras** [comment]
It works. Thanks.

**Comment by sfackler** [comment]
I have a PR open to fix the documentation related to this: #263.
